dev-manuals/architecture/overview.md

```diff
@@ -0,0 +1,28 @@
+# Architecture Overview
+
+The Gamify-IT platform consists of several microservices.
+This document describes the architecture of the platform and its components.
+
+## Table of Contents
+
+<!-- TOC -->
+* [Links](#links)
+* [Components](#components)
+* [Sequence Diagram](#sequence-diagram)
+<!-- TOC -->
+
+## Links
+
+- For information on the specific implementation of the components, see the [dev manuals](https://gamifyit-docs.readthedocs.io/en/latest/dev-manuals/index.html).
```

Comment:
I found no other way than this fixed link. The `index.html` file is generated by readthedocs.